src/serialization/serialization.py

```diff
@@ -0,0 +1,29 @@
+import pickle
+import sys
+import numpy as np
+import font_pb2
+
+def openPickle(foo):
+    f = open(foo, 'rb')
+    cur = pickle.load(f)
+    return cur
+
+
+# read the protibuf with the closed curve information 
+def readSerializedProto(fn):
```

Comment:
This function needs to be documented. Also it needs a better name. You can't export two functions with the same name, for obvious reasons.